src/components/Card.vue

```diff
@@ -0,0 +1,323 @@
+<!--
+  This source file is part of the Swift.org open source project
+
+  Copyright (c) 2022 Apple Inc. and the Swift project authors
+  Licensed under Apache License v2.0 with Runtime Library Exception
+
+  See https://swift.org/LICENSE.txt for license information
+  See https://swift.org/CONTRIBUTORS.txt for Swift project authors
+-->
+
+<template>
+  <Reference
+    class="card"
+    :url="url"
+    :class="classes"
+    v-bind="linkAriaTags"
+  >
+    <CardCover
+      :variants="imageVariants"
+      :rounded="floatingStyle"
+      aria-hidden="true"
+      #default="coverProps"
+    >
+      <slot name="cover" v-bind="coverProps" />
+    </CardCover>
+    <div class="details" aria-hidden="true">
+      <div
+        v-if="eyebrow"
+        :id="eyebrowId"
+        class="eyebrow"
+        :aria-label="formatAriaLabel(`- ${eyebrow}`)"
+      >
+        {{ eyebrow }}
+      </div>
+      <div
+        :id="titleId"
+        class="title"
+      >
+        {{ title }}
+      </div>
+      <ContentNode
```

Comment:
I think you may have a point. That would remove the need for having the async import as well, because if you render a card inside a ContentNode you create an infinite loop.